templates/cassiopeia/scss/_variables.scss

```diff
@@ -77,9 +77,9 @@ $theme-colors: (
 $cassiopeia-toolbar-line-height:         1.8rem;
 
 // Fonts
-$font-family-base:                   "Fira Sans", Arial, sans-serif;
+//$font-family-base:                   "Fira Sans", Arial, sans-serif;
```

Comment:
Is it right that this is commented out? If so, it should be removed instead. Or is it just a remainder from testing?

Reply:
// is a rem

Reply:
@N6REJ Bear I know what a comment is. I ask why it is commented out. Man!!! We shall not have commented out code in our code.

Reply:
The line can be removed. That is a override of the font-family coming from BS4.